### PR TITLE
Jfry feature test pull

### PR DESCRIPTION
**Pull request by jessicafry**
Add jfry folder with files containing favorite books, favorite musicals, and  board game queue.

### CODE REVIEW

File: jfry/books.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Favorite Books: Cat's Cradle, IQ84, Hamlet, Cosmicomics, Sapiens
+Favorite Authors: Vonnegut, Murakami, Kafka, Calvino
+Favorite Genres: SciFi, Literary Fiction, Fantasy
```
Comment (PhiloEpisteme) [inline jfry/books.txt:3]: Love Fantasy

File: jfry/next-game.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Seven Wonders
+Coup
```
Comment (PhiloEpisteme) [inline jfry/next-game.txt]: I think this should move down one on the list.

<reasons>

**Review by PhiloEpisteme — CHANGES_REQUESTED** [review]
Looks good just had one suggestion on game order.

File: jfry/next-game.txt (hunk 2 above)
Comment (jessicafry) [reply jfry/next-game.txt]: changed in latest commit.

**Review by PhiloEpisteme — APPROVED** [review]
looks great!

**Pull request merged**